src/jit/lsra.cpp

```diff
@@ -1755,12 +1756,12 @@ void LinearScan::identifyCandidates()
     unsigned int floatVarCount = 0;
     unsigned int thresholdFPRefCntWtd = 4 * BB_UNITY_WEIGHT;
     unsigned int maybeFPRefCntWtd = 2 * BB_UNITY_WEIGHT;
-#ifdef FEATURE_SIMD
+#if FEATURE_PARTIAL_SIMD_CALLEE_SAVE
```

Comment:
It is very indicative of its purpose. Arm64 has the same issue for their 128-bit vectors - only the lower half is saved across a call. It has not been defined yet for Arm64 because we don't yet support SIMD there. But I presume that we will.